### PR TITLE
chore: zap token error message

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

File: src/views/AddLiquidity/components/ZapErrorMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  if (isSingleToken) return <ZapSingleTokenError />
+  return <ZapPairTokensError />
+})
+
+export const ZapSingleTokenError = memo(() => {
+  const { t } = useTranslation()
+  return (
+    <Message variant="warning" mb="16px">
+      <MessageText>
+        {t('Zap does NOT support tokens with fees on transfers. If you experience any issues, disable Zap here.')}
```
Comment (ChefMomota) [inline src/views/AddLiquidity/components/ZapErrorMessage.tsx:15]: not in translation.json

File: src/views/AddLiquidity/components/ZapErrorMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  )
+})
+
+export const ZapPairTokensError = memo(() => {
+  const { t } = useTranslation()
+  return (
+    <Message variant="warning" mb="16px">
+      <MessageText>
+        {t(
+          `Fee on transfer" tokens should not be used with Limit Orders (use at your own risk)Zap does NOT support tokens with fees on transfers. Please choose "Don't convert" and retry. If you experience any further issues, disable Zap here.`,
```
Comment (ChefMomota) [inline src/views/AddLiquidity/components/ZapErrorMessage.tsx:27]: not in translation.json

File: src/views/AddLiquidity/index.tsx [hunk 3]
```diff
@@ -769,7 +770,7 @@ export default function AddLiquidity() {
                         if (preferZapInstead) {
                           setLiquidityState({
                             attemptingTxn: false,
-                            liquidityErrorMessage: undefined,
+                            liquidityErrorMessage: 'zap error',
```
Comment (chef-jojo) [inline src/views/AddLiquidity/index.tsx:773]: why?

File: src/views/RemoveLiquidity/index.tsx [hunk 4]
```diff
@@ -820,6 +821,7 @@ export default function RemoveLiquidity() {
               </RowBetween>
             )}
           </Box>
+          {error && isZap && <ZapErrorMessages isSingleToken={!(tokenA && tokenB)} />}
```
Comment (chef-jojo) [inline src/views/RemoveLiquidity/index.tsx:824]: use liquidityErrorMessage to detect and put zap error to ConfirmLiquidityModal

**Pull request merged**